docs/howto/create-a-plugin.md

```diff
@@ -0,0 +1,24 @@
+# Creating a Plugin
+After [creating a Workspace](create-a-workspace.md), we can add a Plugin. To do so, use the create-ayx-plugin command. Reply to the prompts and then you will have the template code for your SDK Plugin. For this tool, we use the Input tool type.

```

Comment:
TW review:

After you [create a Workspace](create-a-workspace.md), you can add a Plugin. To do so, use the create-ayx-plugin command. Reply to the prompts and then you will have the template code for your SDK Plugin. For this tool, we use the Input tool type.